pkg/kubectl/describe.go

```diff
@@ -953,6 +960,56 @@ func describeNode(node *api.Node, pods []*api.Pod, events *api.EventList) (strin
 	})
 }
 
+// HorizontalPodAutoscalerDescriber generates information about a horizontal pod autoscaler.
```

Comment:
Worth adding a test for this to [describe_test.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/describe_test.go)?